packages/knip/src/typescript/resolveModuleNames.ts

```diff
@@ -73,11 +69,15 @@ export function createCustomModuleResolver(
     // No need to try and resolve builtins or externals, bail out
     if (isBuiltin(sanitizedSpecifier) || isInNodeModules(name)) return undefined;
 
-    const resolvedFileName = resolveSync(sanitizedSpecifier, containingFile, extensions);
+    const resolvedFileName = resolveSync(sanitizedSpecifier, containingFile, [
+      ...DEFAULT_EXTENSIONS,
+      ...customCompilerExtensions,
+    ]);
```

Comment:
I think this can go back as `extensions` in the `createCustomModuleResolver` scope (slight perf cost to unnecessarily keep re-creating this array with a spread operator).

Reply:
Makes sense and fixed